data/js/server/global.js

```diff
@@ -81,6 +88,12 @@ function onCreatePlayer( pChar )
 
 		TriggerEvent( 8001, "GiveYoungPlayerItems", pChar.socket, pChar );
 	}
+
+	//Attach the special moves Book
+	if( !pChar.npc && !pChar.HasScriptTrigger( 7001 ))
+	{
+		pChar.AddScriptTrigger( 7001 );
+	}
```

Comment:
See above comment about adding uox.ini setting and checking for it here before attaching script to player.